apps/src/StudioApp.js

```diff
@@ -1397,6 +1408,40 @@ StudioApp.prototype.clearHighlighting = function () {
 * @param {FeedbackOptions} options
 */
 StudioApp.prototype.displayFeedback = function (options) {
+  if (experiments.isEnabled('bubbleDialog')) {
+    // eslint-disable-next-line no-unused-vars
+    const { level, response, preventDialog, feedbackType, ...otherOptions } = options;
```

Comment:
Unused var?

Reply:
I don't use `level` here, I'm just using destructuring assignment to check for any unexpected options. Is there an easier way to do that?

Reply:
Ah, that makes sense!